### PR TITLE
chore: bump friendsofphp/php-cs-fixer from 2 => 3

### PR DESCRIPTION
**Pull request by art-cg**
PhpStorm is nagging me about the outdated php-cs-fixer version.

Php 7.1.3 is still supported.
`"php": "^7.1.3 || ^8.0",`
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/composer.json

### CODE REVIEW

**Review by mtarld — CHANGES_REQUESTED** [review]
Thanks for the update!

I think you must update the config filename:
```
Configuration file `.php_cs` is outdated, rename to `.php-cs-fixer.php`.  
```

**Comment by art-cg** [comment]
You are rigth. I need to put more work on this. I saw this after i enabled my Github Actions.
Renaming is not the only task. The content of the file is outdated too.

**Comment by ismail1432** [comment]
Thanks @art-cg 
You can take a look at https://github.com/mtarld/api-platform-ms-bundle/pull/57 I implement changes to pass my PR https://github.com/mtarld/api-platform-ms-bundle/pull/52 😅

**Comment by art-cg** [comment]
@ismail1432 Thanks for the info.
I changed the cache file to the default one and changed it in gitignore too.
I added a little helper entry in composer.json

Test on GitHub Actions looks good. Locally on windows it gives warnings for lineEndings.
`<error severity="warning" source="PHP-CS-Fixer.line_ending" message="Found violation(s) of type: line_ending"/>`
https://github.com/art-cg/api-platform-ms-bundle/runs/4176710611?check_suite_focus=true

File: .php-cs-fixer.php [hunk 1]
```diff
@@ -14,5 +16,5 @@
     ])
     ->setRiskyAllowed(true)
     ->setFinder($finder)
-    ->setCacheFile(__DIR__.'/.php_cs.cache')
-;
+    ->setCacheFile(__DIR__.'/.php-cs-fixer.cache')
+;
```
Comment (mtarld) [inline .php-cs-fixer.php:20]: Missing empty line here

**Review by mtarld — APPROVED** [review]
Just a little review :slightly_smiling_face:

File: .php-cs-fixer.php (hunk 1 above)
Comment (art-cg) [reply .php-cs-fixer.php]: I added the new line at the end

**Comment by mtarld** [comment]
Thank you @art-cg!

**Pull request merged**